src/main/kotlin/dev/stalla/dom/DomParsingExtensions.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("TooManyFunctions")
```

Comment:
Also suppressed @Suppress TooManyFunctions in the extensions, since we have as many extensions as we like.